lib/bundles/inspec-compliance/api.rb

```diff
@@ -5,13 +5,17 @@
 require 'net/http'
 require 'uri'
 
+require_relative 'api/login'
+
 module Compliance
-  class ServerConfigurationMissing < StandardError
-  end
+  class ServerConfigurationMissing < StandardError; end
+  class CannotDetermineServerType < StandardError; end
 
   # API Implementation does not hold any state by itself,
   # everything will be stored in local Configuration store
   class API # rubocop:disable Metrics/ClassLength
+    extend Login
```

Comment:
This may be a matter of taste, but I'd usually have the module name here more deeply namespaced - like Compliance::API::Login .  I don't know that we'd ever (within InSpec) have a name collision with a module named Login, but being specific avoids that possibility.

Reply:
Agreed! Fixed.